### PR TITLE
Review my code

### PR DESCRIPTION
**Pull request by tom7**
okay?

### CODE REVIEW

File: README [hunk 1]
```diff
@@ -0,0 +1,4 @@
+code review!!!!!!
+
+do not submit this!!!
```
Comment (tom7) [inline README:3]: O_O what

Comment (ixchow) [inline README:3]: You should submit this.

File: README [hunk 2]
```diff
@@ -0,0 +1,4 @@
+code review!!!!!!
```
Comment (tom7) [inline README:1]: Welcome to my code review!
no action required

**Review by tom7 — COMMENTED** [review]
I think this is a good change......... but I made it ;-)

File: README (hunk 1 above)
Comment (tom7) [reply README:3]: no

Comment (ixchow) [reply README:3]: why not

**Comment by dwrensha** [comment]
This change is bad and you should feel bad.

**Review by dwrensha — CHANGES_REQUESTED** [review]
I request changes.

**Comment by tom7** [comment]
just going for it 😎

**Comment by tom7** [comment]
> I request changes.

Sorry.......already submitted 

- [ ] apologize
- [ ] check off this item

**Pull request merged**